src/helpers/index.ts

```diff
@@ -21,18 +21,13 @@ export const getDEC = (hex: string): number => {
 
 //---Convert to hexadecimal
 export const getHEX = (number: NumberOrString): string => {
-    const hex = round(number).toString(16).toUpperCase();
-    if (hex.length === 1) { return `0x0${hex}`; }
+    const hex = round(number).toString(16).toUpperCase().padStart(2,'0');
```

Comment:
The introduction of `padStart` is a breaking change for those using old `Node` versions. This would require to set a minimum `Node` version in the package and release it in a major version, but it doesn‘t worth it because the change is just aesthetic.